storage/queue_rocksdb.go

```diff
@@ -121,7 +121,7 @@ func (q *rocksQueue) Page(start int64, count int64, fn func(index int, k, v []by
 		if err != nil {
 			return err
 		}
-		count--
```

Comment:
This kind of change is purely syntactical and should be avoided.  I believe -- is used because golint prefers it.

Reply:
I just applied this changes because this commit: https://github.com/contribsys/faktory/commit/c48093afb123701eaeb92b11751cc88d8fd20767